packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -221,14 +209,6 @@ contract TaikoL1 is
         });
     }
 
-    /**
-     * Check if the L1 is halted.
-     * @return True if halted, false otherwise.
-     */
-    function isHalted() public view returns (bool) {
```

Comment:
Halting based some scenario's still a good idea I think if it doesn't add too much complexity to the protocol, but probably good to remove for the testnet anyway because it could be abused.

Reply:
Agreed, but lets add that feature later. I figured it's a mistake to complicate the design/impl before a simpler version has been well tested.